src/main/java/frc/robot/commands/EjectGP.java

```diff
@@ -5,23 +5,30 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.Constants;
+
+import frc.robot.Constants.constLED;
+import frc.robot.subsystems.Hopper;
 import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.LED;
 
 public class EjectGP extends Command {
   /** Creates a new EjectGP. */
-
+  LED globalLED;
   Intake globalIntake;
+  Hopper globalHopper;
 
-  public EjectGP(Intake subIntake) {
+  public EjectGP(Intake subIntake, Hopper subHopper, LED ejectLED) {
     // Use addRequirements() here to declare subsystem dependencies.
+    this.globalHopper = subHopper;
     this.globalIntake = subIntake;
+    globalLED = ejectLED;
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    globalIntake.setIntakeVelocity(Constants.constIntake.INTAKE_EJECT_VELOCITY);
+    globalIntake.setIntakeVelocity(-0.5);
```

Comment:
replace this line with globalIntake.setIntakeVelocity(Constants.constIntake.INTAKE_EJECT_VELOCITY)